### PR TITLE
Remove jQuery from web-based installer

### PR DESCRIPTION
**Pull request by rrrnld**
Removes the jQuery dependency to fix the problem described in #3387.

### CODE REVIEW

**Comment by heyarne** [comment]
I could unfortunately only test this via `python2 dokku-installer.py`. More thorough testing (i.e. a complete setup process) would be required. The validation logic seems to work fine, I'm not sure about the server response handling.

The downside of this is that jQuery offers greater browser support. The most modern feature I used is [`fetch`](https://caniuse.com/#search=fetch) which is supported by all modern browsers.

**Comment by josegonzalez** [comment]
Seems fine. Can you try this out on a local vagrant-vm setup, as per our [docs](http://dokku.viewdocs.io/dokku/getting-started/install/vagrant/)? Use your branch and run `vagrant up` to get your changes going.

File: contrib/dokku-installer.py [hunk 1]
```diff
@@ -340,7 +340,8 @@ def main():
             window.location.href = "http://dokku.viewdocs.io/dokku~{VERSION}/deployment/application-deployment/";
           }, 3000);
         })
-        .catch(function () {
+        .catch(function (error) {
+          console.error(error)
```
Comment (josegonzalez) [inline contrib/dokku-installer.py]: We should avoid having any console logging.

Comment (josegonzalez) [reply contrib/dokku-installer.py]: Please do :)

**Comment by heyarne** [comment]
To debugs this. I need to the server's log ouput; there doesn't seem to be anything relevant in `/var/logs`, can you tell me where to find it?

**Comment by josegonzalez** [comment]
It doesn't log anything anywhere. I'd just run it locally and make sure the SSH keys etc. are setup properly.

**Comment by heyarne** [comment]
Sooo... this took a bit longer than expected, but there you go. This is now tested inside vagrant and works, hope this helps! Additionally I added a `<meta charset="utf-8" />` to remove the warning about a missing charset declaration. The setup now loads without any issues or warnings inside the Firefox console.

**Pull request merged**